tests/robustness/traffic/kubernetes.go

```diff
@@ -133,7 +139,19 @@ func (t kubernetesTraffic) RunTrafficLoop(ctx context.Context, c *client.Recordi
 func (t kubernetesTraffic) Read(ctx context.Context, c *client.RecordingClient, s *storage, limiter *rate.Limiter, keyPrefix string) error {
 	kc := kubernetes.Client{Client: &clientv3.Client{KV: c}}
 	op := random.PickRandom(t.readChoices)
+	var rev int64
 	switch op {
+	case KubernetesGetStale:
+		_, rev = s.PickRandom()
+		fallthrough
```

Comment:
Let's be simple and direct.

Reply:
I hope that use of `cmp.Or` isn't too obscure (I want to use `rev` from a different `key` if possible)

Reply:
It is, can you just implement a method that you need in the store struct?

Reply:
Good idea. I didn't think I should change that. Added `KeyWithUnrelatedRev` to `storage`. Looks much better (except the name).

Thanks